src/main/java/duckie/command/Command.java

```diff
@@ -0,0 +1,30 @@
+package duckie.command;
+
+import duckie.task.TaskList;
+import duckie.Ui;
+import duckie.Storage;
+import duckie.exception.DuckieException;
+
+/**
+ * Parent class of all the Commands action
+ */
+public abstract class Command {
+
+    /**
+     * Execute the command according to the input given
+     * @param tasks TaskList containing all the tasks
+     * @param ui Ui to interact with the users
+     * @param storage Storage to write to File
```

Comment:
Should there be a fullstop behind each param line because "Punctuation behind each parameter description"